src/Console/EasyCodingStandardConsoleApplication.php

```diff
@@ -56,6 +58,14 @@ public function doRun(InputInterface $input, OutputInterface $output): int
             unset($xdebugHandler);
         }
 
+        if ($input->hasParameterOption(['--version', '-V'], true) === true) {
+            $output->writeln($this->getLongVersion());
+            $output->writeln(sprintf('- %s <info>%s</info>', 'PHP_CodeSniffer', PHP_CodeSniffer::VERSION));
+            $output->writeln(sprintf('- %s <info>%s</info>', 'PHP-CS-Fixer', PhpCsFixer::VERSION));
+
+            return 0;
+        }
```

Comment:
Overrides the default `--version`.

https://github.com/symfony/console/blob/c981e0e9380ce9f146416bde3150c79197ce9986/Application.php#L223-L227